tike/tike.py

```diff
@@ -83,11 +85,13 @@ def _combined_interface(obj,
     return None
 
 
-def admm(obj=None, voxelsize=1.0,
-         data=None,
-         probe=None, theta=None, h=None, v=None, energy=None,
-         niter=1, rho=0.5, gamma=0.25,
-         **kwargs):
+def admm(
+    obj=None, voxelsize=1.0,
+    data=None,
```

Comment:
I'd reserve the first parameter to data.

Reply:
Reordering parameters is API breaking, so it is out of the scope of this pull.

Reply:
I've opened #18 to address this request.